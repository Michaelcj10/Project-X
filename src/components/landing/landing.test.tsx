import Landing from "./landing";
import * as React from "react";
import { render } from "@testing-library/react";

describe("<Landing /> spec", () => {
  it("renders the component", () => {
    const container = render(<Landing />);
    expect(container).toMatchSnapshot();
  });
});
