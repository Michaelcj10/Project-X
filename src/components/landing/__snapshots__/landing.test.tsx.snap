// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing /> spec renders the component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
        <div
            class="sc-AykKE lbMCaW"
        >
            <div
                class="hero-block"
            >
                <svg
                    xmlns="http://www.w3.org/2000/svg"
                    viewBox="0 0 1440 320"
                >
                    <path
                        fill="#fff"
                        fill-opacity="1"
                        d="M0,64L34.3,74.7C68.6,85,137,107,206,112C274.3,117,343,107,411,117.3C480,128,549,160,617,192C685.7,224,754,256,823,245.3C891.4,235,960,181,1029,170.7C1097.1,160,1166,192,1234,197.3C1302.9,203,1371,181,1406,170.7L1440,160L1440,320L1405.7,320C1371.4,320,1303,320,1234,320C1165.7,320,1097,320,1029,320C960,320,891,320,823,320C754.3,320,686,320,617,320C548.6,320,480,320,411,320C342.9,320,274,320,206,320C137.1,320,69,320,34,320L0,320Z"
                    />
                </svg>
                <svg
                    xmlns="http://www.w3.org/2000/svg"
                    viewBox="0 0 1440 320"
                >
                    <path
                        fill="#fff"
                        fill-opacity="1"
                        d="M0,160L30,160C60,160,120,160,180,144C240,128,300,96,360,69.3C420,43,480,21,540,48C600,75,660,149,720,170.7C780,192,840,160,900,138.7C960,117,1020,107,1080,117.3C1140,128,1200,160,1260,149.3C1320,139,1380,85,1410,58.7L1440,32L1440,320L1410,320C1380,320,1320,320,1260,320C1200,320,1140,320,1080,320C1020,320,960,320,900,320C840,320,780,320,720,320C660,320,600,320,540,320C480,320,420,320,360,320C300,320,240,320,180,320C120,320,60,320,30,320L0,320Z"
                    />
                </svg>
                <div
                    class="row"
                >
                    <div
                        class="six columns centered"
                    >
                        <div>
                            <h1>
                                Mad Goat Studios
                            </h1>
                        </div>
                        <div>
                            <p>
                                Website and application development with a difference.
                            </p>
                        </div>
                        <div>
                            <button
                                class="sc-AykKC bXgdYi"
                            >
                                Learn more
                            </button>
                        </div>
                    </div>
                    <div
                        class="six columns centered"
                    >
                        <div
                            class="sc-AykKD iIeLAL"
                        >
                            <div
                                class="css-1rcyed2"
                            >
                                <span>
                                    <span
                                        class="css-19bon8n react-loading-skeleton"
                                        style="width: 0px; height: 0px;"
                                    >
                                        ‌
                                    </span>
                                </span>
                            </div>
                        </div>
                        <img
                            style="visibility: hidden;"
                            src="mobile-2.svg"
                            alt="Landing image"
                        />
                    </div>
                </div>
            </div>
        </div>
    </div>
</body>,
  "container": <div>
    <div
        class="sc-AykKE lbMCaW"
    >
        <div
            class="hero-block"
        >
            <svg
                xmlns="http://www.w3.org/2000/svg"
                viewBox="0 0 1440 320"
            >
                <path
                    fill="#fff"
                    fill-opacity="1"
                    d="M0,64L34.3,74.7C68.6,85,137,107,206,112C274.3,117,343,107,411,117.3C480,128,549,160,617,192C685.7,224,754,256,823,245.3C891.4,235,960,181,1029,170.7C1097.1,160,1166,192,1234,197.3C1302.9,203,1371,181,1406,170.7L1440,160L1440,320L1405.7,320C1371.4,320,1303,320,1234,320C1165.7,320,1097,320,1029,320C960,320,891,320,823,320C754.3,320,686,320,617,320C548.6,320,480,320,411,320C342.9,320,274,320,206,320C137.1,320,69,320,34,320L0,320Z"
                />
            </svg>
            <svg
                xmlns="http://www.w3.org/2000/svg"
                viewBox="0 0 1440 320"
            >
                <path
                    fill="#fff"
                    fill-opacity="1"
                    d="M0,160L30,160C60,160,120,160,180,144C240,128,300,96,360,69.3C420,43,480,21,540,48C600,75,660,149,720,170.7C780,192,840,160,900,138.7C960,117,1020,107,1080,117.3C1140,128,1200,160,1260,149.3C1320,139,1380,85,1410,58.7L1440,32L1440,320L1410,320C1380,320,1320,320,1260,320C1200,320,1140,320,1080,320C1020,320,960,320,900,320C840,320,780,320,720,320C660,320,600,320,540,320C480,320,420,320,360,320C300,320,240,320,180,320C120,320,60,320,30,320L0,320Z"
                />
            </svg>
            <div
                class="row"
            >
                <div
                    class="six columns centered"
                >
                    <div>
                        <h1>
                            Mad Goat Studios
                        </h1>
                    </div>
                    <div>
                        <p>
                            Website and application development with a difference.
                        </p>
                    </div>
                    <div>
                        <button
                            class="sc-AykKC bXgdYi"
                        >
                            Learn more
                        </button>
                    </div>
                </div>
                <div
                    class="six columns centered"
                >
                    <div
                        class="sc-AykKD iIeLAL"
                    >
                        <div
                            class="css-1rcyed2"
                        >
                            <span>
                                <span
                                    class="css-19bon8n react-loading-skeleton"
                                    style="width: 0px; height: 0px;"
                                >
                                    ‌
                                </span>
                            </span>
                        </div>
                    </div>
                    <img
                        style="visibility: hidden;"
                        src="mobile-2.svg"
                        alt="Landing image"
                    />
                </div>
            </div>
        </div>
    </div>
</div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
